controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -238,17 +259,51 @@ func (r *StorageClassRequestReconciler) initPhase(storageProfile *v1.StorageProf
 			// if we found more than one CephBlockPool, we can't determine which one to select, so error out
 			cbpItemsLen := len(cephBlockPoolList.Items)
 			if cbpItemsLen == 0 {
-				cbpNewName := fmt.Sprintf("cephblockpool-%s-%s", r.storageConsumer.Name, generateUUID())
+				cbpNewName := fmt.Sprintf("cephblockpool-profile-%s", storageProfile.Name)
```

Comment:
am I wrong in assuming that we still need to create storageprofile resource even after this PR gets merged?

Reply:
Anything StorageProfile related will be removed in a subsequent commit. Currently debating whether to do it in another PR or delay this PR further to wait for it.